target/linux/ath79/dts/qca9531_glinet_gl-ar300m.dtsi

```diff
@@ -1,3 +1,5 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
This license change will require everyone to ACK, won't it? I do ack, but there are others...
Think about removing this change from your PR, as it will be just another obstactle.

(Same applies for lite.dts. There you are at least the only person having changed that particular file, but it emerged out of another DTS, so you have the same argument again.)

Reply:
I've reviewed the history of the various files that add "OR MIT" to the licensing and, other than this one, they represent new content or content for which I was the author.

I'll address this one by removing the license change at this time.